django_informixdb/base.py

```diff
@@ -244,6 +244,23 @@ def get_new_connection(self, conn_params):
         self.connection.add_output_converter(pyodbc.SQL_LONGVARCHAR, self._output_converter)
         self.connection.add_output_converter(pyodbc.SQL_WLONGVARCHAR, self._output_converter)
 
+        # This will set database LOCK MODE WAIT at connection level
+        # Application can use this property to override the default server process for accessing a locked row or table.
+        # The default value is 0 (do not wait for the lock).
+        # Possible values:
+        #    -1 - WAIT until the lock is released.
+        #    0 - DO NOT WAIT, end the operation, and return with error.
+        #    nn - WAIT for nn seconds for the lock to be released.
+        if 'LOCK_MODE_WAIT' in conn_params['OPTIONS']:
+            if conn_params['OPTIONS']['LOCK_MODE_WAIT']:
+                wait_parameter = conn_params['OPTIONS']['LOCK_MODE_WAIT']
+                if wait_parameter == -1:
+                    wait_parameter = ''
+                sql = 'SET LOCK MODE TO WAIT ' + str(wait_parameter)
```

Comment:
A safer line here would be:

    sql = 'SET LOCK MODE TO WAIT %d' % wait_parameter

as that will also type-check the parameter to ensure it's an integer, hopefully preventing accidental weird configs.

Reply:
This will not handle all possible situation. So keeping it as is.